samples/property/provider/src/main.rs

```diff
@@ -145,6 +121,63 @@ fn start_ambient_air_temperature_data_stream(subscription_map: Arc<Mutex<Subscri
     });
 }
 
+/// Publish a message to a MQTT broker located.
+///
+/// # Arguments
+/// `host` - The MQTT broker's URI.
+/// `topic` - The topic to publish to.
+/// `content` - The message to publish.
+fn publish_message(host: &str, topic: &str, content: &str) -> Result<(), String> {
+    let create_opts = mqtt::CreateOptionsBuilder::new()
+        .server_uri(host)
+        .client_id(MQTT_CLIENT_ID.to_string())
+        .finalize();
+
+    let client = mqtt::Client::new(create_opts)
+        .map_err(|err| format!("Failed to create the client due to '{:?}'", err))?;
+
+    let conn_opts = mqtt::ConnectOptionsBuilder::new()
+        .keep_alive_interval(Duration::from_secs(30))
+        .clean_session(true)
+        .finalize();
+
+    let _connect_response =
+        client.connect(conn_opts).map_err(|err| format!("Failed to connect due to '{:?}", err));
+
+    let msg = mqtt::Message::new(topic, content, MQTT_QOS);
+    if let Err(err) = client.publish(msg) {
+        return Err(format!("Failed to publish message due to '{:?}", err));
```

Comment:
return Err(format!("Failed to publish message due to '{err:?}'"));

Reply:
Fixed.